config/locales/server.en.yml

```
@@ -12,6 +12,7 @@ en:
     code_review_unmerged_tag: "Tag to apply to unmerged commits"
     code_review_commit_tag: "Tag to apply to commit topics"
     code_review_pull_request_tag: "Tag to apply to pull request topics"
+    code_review_create_admin_only_categories: "Cause new code review categories to be viewable only by admins"
```

```suggestion
    code_review_create_admin_only_categories: "New code review categories may only be viewed by admins"
```
Wasn't crazy on starting the description with "Cause"